packages/cli/src/install/install.js

```diff
@@ -19,13 +19,13 @@ const spawnOpts = {
 function install(args, ctx) {
   const name = args[0];
 
-  let res = new PackageManager({ projectDir: ctx.root }).install([name]);
-
-  if (res.status) {
-    process.exit(res.status);
+  try {
+    new PackageManager({ projectDir: ctx.root }).install([name]);
```

Comment:
No need to do it. All you need to do is to make `install` an `async function` and `try/catch` we have in `cliEntry.js` will log the error automatically.